src/convertToObject.js

```diff
@@ -6,7 +6,40 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resObj = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `resObj`. It will decrease code readability. Consider using a more descriptive name, such as `styleObject` or `cssProperties`.